### DISCUSSION TITLE
Releases for Supported Boards

### DISCUSSION THREAD

**Discussion by KozhinovAlexander** [CANnectivity/cannectivity#37] [Ideas]
It would be very helpful to build in CI binaries for already supported boards, s.t. a user can be ready to go within few clicks.

**Comment by henrikbrixandersen** [comment]
> It would be very helpful to build in CI binaries for already supported boards, s.t. a user can be ready to go within few clicks.

Given that every board supported by Zephyr, which has CAN and USB device support, is supported by CANnectivity, that would be quite a few boards. This is not feasible.

**Reply by KozhinovAlexander** [reply]
What if the number of bords will be limited to at most 10 for example. Those boards would be specific designed boards for can-usb protocol conversion like yours USB2CAN for example.

**Reply by henrikbrixandersen** [reply]
No. That could easily give the impression that only these boards are supported. 

I have no plans for hosting binaries of CANnectivity.
